apps/sync/test/unit/bootstrap_test.js

```diff
@@ -0,0 +1,182 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+/* global
+  DataAdapters,
+  ERROR_SYNC_APP_SYNC_IN_PROGRESS,
+  ERROR_SYNC_INVALID_REQUEST_OPTIONS,
+  expect,
+  Bootstrap,
+  BootstrapFixture,
+  LazyLoader,
+  require,
+  requireApp,
+  suite,
+  SyncEngineMock,
+  test
+*/
+
+
+require('/shared/js/sync/errors.js');
+requireApp('sync/test/unit/fixtures/bootstrap.js');
+requireApp('sync/test/unit/sync-engine-mock.js');
+requireApp('sync/test/unit/test-adapter-mock.js');
+require('/shared/test/unit/mocks/mock_lazy_loader.js');
+requireApp('sync/test/unit/lazy-loader-mock.js');
```

Comment:
Can't you extend shared/test/unit/mocks/mock_lazy_loader.js and get rid of this mock?